docs/migrations/v2-to-v3.md

```diff
@@ -55,6 +55,32 @@ app.UpgradeKeeper.SetUpgradeHandler("v3",
 The host and controller submodule params only need to be set if you integrate those submodules. 
 For example, if a chain chooses not to integrate a controller submodule, it does not need to set the controller params. 
 
+### Genesis migrations
+
+If the chain will adopt ICS27 and chooses to upgrade via a genesis export, then the ICS27 parameters must be set during genesis migration. 
+
+The migration code required may look like:
+
+```go
+    controllerGenesisState := icatypes.DefaultControllerGenesis()
+    // overwrite parameters as desired
+    controllerGenesisState.Params = icacontrollertypes.Params{
+        ControllerEnabled: true, 
+    } 
+
+    hostGenesisState := icatypes.DefaultHostGenesis()
+    // overwrite parameters as desired
+    hostGenesisState.Params = icahosttypes.Params{
+        HostEnabled: true, 
+        AllowMessages: []string{"/cosmos.bank.v1beta1.MsgSend", ...], 
+    }
+
+    icaGenesisState := icatypes.NewGenesisState(controllerGenesisState, hostGenesisState)
+
+    // set new ics27 genesis state
+	appState[icatypes.ModuleName] = clientCtx.JSONCodec.MustMarshalJSON(icaGenesisState)
```

Comment:
indentation nit: I think the suggested comment is right, but maybe not, you may need to do it locally. 

```suggestion
    appState[icatypes.ModuleName] = clientCtx.JSONCodec.MustMarshalJSON(icaGenesisState)
```